src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -618,7 +618,7 @@ private static async Task<IEnumerable<TextChange>> GetTextChangesFromTextDiffere
 
                 foreach (var replacement in relevantReplacements)
                 {
-                    var buffer = snapshotSpanToClone.HasValue ? textBufferCloneService.Clone(snapshotSpanToClone.Value) : _textBufferFactoryService.CreateTextBuffer(preMergeDocumentTextString, contentType);
+                    var buffer = snapshotSpanToClone.HasValue ? textBufferCloneService.CloneWithUnknownContentType(snapshotSpanToClone.Value) : _textBufferFactoryService.CreateTextBuffer(preMergeDocumentTextString, contentType);
```

Comment:
Although I've been meaning to delete this code for some time, what was the reason for the change?

Reply:
when we clone, we sometimes clone it with unknown content type to make sure, the buffer is used only internally (no buffer listener or when attached to view, view listener associated with certain content type to be created) - I dont know who decide this, but someone decided so I am following that with explicit method name since I am now also cloning buffer with certain content type to use it with view and etc.